content/millennium/r4/clinical/medications/medication-administration.md

```diff
@@ -48,42 +58,44 @@ The following fields are returned if valued:
-
 ### Authorization Types
 
 <%= authorization_types(provider: true, patient: false, system: true) %>
 
 ### Parameters
 
- Name                   | Required?          | Type          | Description
-------------------------|--------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
- `_id`            | This, or `patient` | [`token`]     | The logical resource id associated with the resource. Example: `_id=12345`
- `patient`        | This, or `_id`     | [`reference`] | The specific patient to return MedicationAdministrations for. Example: `patient=1234`
- `status`         | N                  | [`token`]     | The [status] of the medication administration event. May be a list separated by commas. Example: `status=completed,not-done`
- `performer`      | N                  | [`reference`] | The identity of the individual who administered the medication. May be a list separated by commas. Example: `performer=1245,7659`
- `effective-time` | N                  | [`date`]      | An date-time range when administration happened. Must include a time. Example: `effective-time=ge2014-05-19T20:54:02.000Z`
- `_count`         | N                  | [`number`]    | The maximum number of results to include in a page. Example: `50`
+ Name             | Required?     | Type          | Description
+------------------|---------------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`            | Conditionally | [`token`]     | The logical resource id associated with the resource. It may be a list separated by commas. This parameter is required if the `patient` parameter is not used. Example: `_id=12345`
+ `patient`        | Conditionally | [`reference`] | The specific patient to return medication administrations for. This parameter is required if the `_id` parameter is not used. Example: `patient=1234`
+ `status`         | No            | [`token`]     | The status of the medication administration event. It may be a list separated by commas. Example: `status=completed,not-done`
+ `performer`      | No            | [`reference`] | The identity of the individual who administered the medication. It may be a list separated by commas. Example: `performer=1245,7659`
+ `effective-time` | No            | [`date`]      | A date range for when the administration happened. Example: `effective-time=ge2014-05-19T20:54:02.000Z`
+ `_count`         | No            | [`number`]    | The maximum number of results to include in a page. Example: `50`
```

Comment:
add a link to _count like other resources
http://hl7.org/fhir/R4/search.html#count